explainer.md

```diff
@@ -371,26 +337,110 @@ if(frameOfRef.bounds) {
-});
+    // Request the next window callback
+    window.requestAnimationFrame(onDrawFrame);
+  }
+}
+
+function drawMultiviewScene(sequence<VRView> views, VRPose pose) {
+  for (let view in views) {
+    let viewMatrix = pose.getViewMatrix(view);
+    let projectionMatrix = view.projectionMatrix;
+    
+    // Set uniforms as appropriate for shaders being used
+  }
+  
+  // Draw Scene
+}
+
+function drawScene(VRView view, VRPose pose) {
+  let viewMatrix = null;
+  let projectionMatrix = null;
+  if (view) {
+    viewMatrix = pose.getViewMatrix(view);
+    projectionMatrix = view.projectionMatrix;
+  } else {
+    viewMatrix = defaultViewMatrix;
+    projectionMatrix = defaultProjectionMatrix;
+  }
+  
+  // Set uniforms as appropriate for shaders being used
+  
+  // Draw Scene
+}
+```
+
+### High quality rendering
+While in exclusive sessions, the UA is responsible for providing a framebuffer that is correctly optimized for presentation to the `VRSession` in each VRFrame. Developers can optionally request either the buffer size or viewport size be scaled, though the UA may not respect the request. Even when the UA honors the scaling requests, the result is not guaranteed to be the exact percentage requested.
+
+The first scaling mechanism is done by specifying a `framebufferScaleFactor` at `VRWebGLLayer` creation time. In response, the UA may create a framebuffer that is based on the requested percentage of the maximum size supported by the `VRDevice`. On some platforms such as Daydream, the UA may set the default value of `framebufferScaleFactor` to be less 1.0 for performance reasons. Developers explicitly wishing to use the full resolution on these devices can do so by requesting the `framebufferScaleFactor` be set to 1.0. 
+
+```js
+function setupWebGLLayer() {
+  return vrDevice.ensureContextCompatibility(gl).then(() => {
+    vrSession.baseLayer = new VRWebGLLayer(vrSession, glCanvas, { framebufferScaleFactor:0.8 });
+  });
+```
+
+The second scaling mechanism is to request a scaled viewport into the `VRWebGLLayer.framebuffer`. For example, under times of heavy load the developer may choose to temporarily render fewer pixels. To do so, developers should call `VRWebGLLayer.requestViewportScaling()` and supply a value between 0.0 and 1.0. The UA may then respond by changing the `VRWebGLLayerTarget.framebuffer` and/or the `VRWebGLViewport` values in future VR rendering frames. It is worth noting that the UA may change the viewports for reasons other than developer request; as such, developers must always query the viewport values on each VR rendering frame.
+
+```js
+function onDrawFrame() {
+  // In response to a performance dip, request the viewport be restricted 
+  // to a percentage (ex: 50%) of the layer's actual buffer. This change
+  // will apply to subsequent rendering frames
+  layer.requestViewportScaling(0.5);
+  
+  // Register for next frame callback
+  vrSession.requestVRPresentationFrame().then(onDrawFrame);
```

Comment:
By putting this call in the frame's promise handler, I was trying to illustrate that this is something that's ok (and somewhat encouraged) to change regularly.  What about this:
```js
function onDrawFrame() { 
  // In response to a performance dip, request the viewport be restricted  
  // to a percentage (ex: 50%) of the layer's actual buffer. This change 
  // will apply to subsequent rendering frames 
  layer.requestViewportScaling(0.5); 

  // Draw the current frame 
  
  // Register for next frame callback 
  vrSession.requestVRPresentationFrame().then(onDrawFrame); 
}
```